### DISCUSSION TITLE
Bug Report: Unexpected crash through specific steps.

### DISCUSSION THREAD

**Discussion by DinoAwakening** [keiwando/toonsquid#282] [Q&A]
iPad Air 5, M1. (16.4.1)

Select and area from a drawing layer, delete it with the trash can, copy a drawing layer from the next frame, paste it above the drawing we had deleted, merge. It results in crash every time. 

(Changing the steps order may cause the app to not crash and having the expected result)

Edit: Some additional conditions are that before dropping de new drawing, I merge it down manually. Also, the layer is above a “group”, so it might be problematic aswell. My theory is that AutoMerge tries to merge down my layer already merged, but with the “group” causing a fatal error.

**Comment by keiwando** [comment]
Thanks for reporting this! I will look into it.
